README.md

```diff
@@ -2,28 +2,49 @@
 
 # Amsterdam Design System
 
-[Demo site with the storybook of the components](https://amsterdam.github.io/design-system/)
+This is the guide for the user interfaces of all digital products and services provided by the City of Amsterdam.
+
+We offer reusable components, patterns and guides for teams that work on our websites and applications.
+This helps everyone work faster and better, with more time to create value.
+To our citizens and businesses, all our channels look the same and work similarly;
+they are broadly accessible and evoke trust.
+
+We aim to create libraries for, or support otherwise, Figma, CSS, React, React Native, Salesforce, and Mendix.
+
+## Current state
+
+We have made progress with foundations such as a grid and typography. Additionally, around 20 (MVP) components are available, first focusing on creating a homepage. We will continue to develop components and write documentation to help more teams benefit and process feedback on our offering.
```

Comment:
Do we want to add this to the README? It means we have to continually and manually update it

Reply:
Done